update/Pos/Update/Poll/Types.hs

```diff
@@ -211,6 +213,9 @@ data PollModifier = PollModifier
     , pmActiveProps       :: !(MapModifier UpId ProposalState)
     , pmDelActivePropsIdx :: !(HashMap ApplicationName (HashSet UpId))
     , pmSlottingData      :: !(Maybe SlottingData)
+    , pmEpochProposers    :: !(Maybe (HashSet StakeholderId))
+    -- ^ Cache of DB state.
```

Comment:
@gromakovsky 

>As a consequence, if we don't modify epoch proposers, but only read them, we will still write proposers to DB. AFAIU, it's impossible now, so it's not important.

we can add flag later, no problems

>· · I don't like that you are mixing actual GState (well, cache of this state) and modifier in one type.

I more or less agree  with you. Do you propose to split them on two different datas in this PR?
I think it should be done in the future.

actually, when we will have GState in memory, it shouldn't be cache, we must load it on start. so it's kostil now.